### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by fr1k** [ChrisAnd1998/TaskbarX#426]
The application hangs in the process and the error in the logs.
In terminal:
The "instance object [Taskbar] Accessibility" method is missing.Iaccisible::get_accRole(object)" from the "System.Int32" class"
![image](https://user-images.githubusercontent.com/788919/95650642-42958c00-0afe-11eb-9b1e-a7b978cb8b1c.png)

in 1.6.1.0 bug too

Windows 10 Pro
19041.508

**Comment by fr1k** [comment]
in dnSpy :
![image](https://user-images.githubusercontent.com/788919/95650605-137f1a80-0afe-11eb-928f-6ff38b291396.png)

**Comment by fr1k** [comment]
the bug was already there
